tests/ecosystem/destroy/test_destroy_cluster.py

```diff
@@ -1,14 +1,21 @@
 import logging
 
-from ocs_ci.framework import config
 from ocs_ci.framework.testlib import EcosystemTest, destroy
-from ocs_ci.utility.utils import destroy_cluster
+from ocs_ci.framework import config
 
 log = logging.getLogger(__name__)
 
 
 @destroy
 class TestDestroy(EcosystemTest):
     def test_destroy_cluster(self, log_cli_level):
-        log.info("Running OCS cluster destroy")
-        destroy_cluster(config.ENV_DATA['cluster_path'], log_cli_level)
+        teardown = config.RUN['cli_params'].get('teardown')
```

Comment:
the cli param is 'destroy' or 'teardown'?

Reply:
@RazTamir the CLI param is `--teardown`[1] 

[1] https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/ocscilib.py#L58,L64

Reply:
so I think this https://github.com/red-hat-storage/ocs-ci/pull/705/files#diff-4be2078a125fe364f7198675d51bc688R135 should stay 'deployment' and not 'destroy'.
We are tearing down the deployment

Reply:
@RazTamir I understand your linguistic point here, the problem is, that with `deployment` it will launch the `test_cluster_is_running` test, which is quite ok for properly running cluster (the test passes and during teardown phase it will destroy the cluster), but for partially working cluster, the *test part* of the `test_cluster_is_running` fails and even though the cluster is properly destroyed in the teardown phase, the overall status is failed. This is not big problem for manual work, because it is easily visible, that the cluster was clearly destroyed and almost nobody is checking the return code, but it is real problem in CI, because it is not possible to distinguish if the cluster was properly destroyed and we can continue with the cleanup of the env. or not.

So I'm not sure, what would be the best solution here? Or would be enough to change the `destroy` marker to `teardown`, or on the other hand change the `--teardown` parameter to `--destroy`? The later one will be slightly bigger change affecting whole team, as everybody is using it.

Reply:
ok.
Let's not complicate things now.
We can revisit this in the future

Reply:
Here is another Idea, how to do it and for me it seems to be the best and cleanest one:

- rename the `test_cluster_is_running` from `test_ocs_basic_install.py` to something like `test_deployment`, so it will not be directly connected only with installation (it will stay marked as `deployment` as it is now)
- The `test_deployment` will run the `assert is_cluster_running(...)` only if `--destroy` parameter will not be specified. So there will be no change in the behaviour for installation as it is now (except that the test will not be named `test_cluster_is_running` but `test_deployment`). But for teardown, it will not check the cluster availability and thus not fails on cluster which is not fully installed.

- Then we can completely remove the `test_destroy_cluster` test.

Reply:
Yep, sounds like good solution. ACK from my side to go with this approach.

Reply:
+1,
looks much better

Reply:
Will close this PR and create new one with the new approach.